README.md

```diff
@@ -8,9 +8,24 @@ Serve content read from a Libero API to the public.
 Getting started
 ---------------
 
-To run a website reading from two content services (`blog-articles` and `scholarly-articles`):
+1. Run `docker-compose down --volumes --remove-orphans && docker-compose up --build`.
 
-1. Create `config/packages/content_page.yaml`:
+2. Open http://localhost:8080/articles/article1 to see the scholarly article `article1` in the `dev` environment.
+
+### Configuration
+
+The `dev` environment reads content from a [Libero dummy API](https://github.com/libero/dummy-api), using the configuration and data in [`.docker/api`](.docker/api).
+
+Running in production
```

Comment:
inconsistent title format `---` vs `###`?

Reply:
There's underline styles only for levels 1 and 2.

Reply:
But why? :thinking: